app-scroll-behaviors/app-custom-scroller-behavior.html

```diff
@@ -38,26 +38,23 @@
        *
        *```html
        * <div id="scrollingRegion" style="overflow-y: auto;">
-       *  <app-header scroller="#scrollingRegion">
+       *  <app-header scroll-target="scrollingRegion">
        *  </app-header>
        * </div>
        *```
-       * In this case, `scroller` will point to the outer div element. Alternatively, 
+       * In this case, `scrollTarget` will point to the outer div element. Alternatively, 
        * you can set the property programatically:
        *
        *```js
-       * appHeader.scroller = document.querySelector("#scrollingRegion");
+       * appHeader.scrollTarget = document.querySelector("#scrollingRegion");
        *```
        *
-       * ### Window (default)
-       * The scroller property can also point to `window` to use . e.g.
-       *```html
-       * <app-header scroller="window"></app-header>
-       *```
-       * Alternatively, you can set the property programatically:
+       * ### Document Element 
+       * By default the `scrollTarget` property points to the document element. Alternatively,
+       * you can set the property programatically:
        *
        *```js
-       * appHeader.scroller = window;
+       * appHeader.scrollTarget = document.documentElement;
```

Comment:
We don't need this behavior anymore.  Let's remove this file and probably the `app-scroll-behaviors` folder as well.